lib/fitbank_api/pix/payment_order.rb

```diff
@@ -0,0 +1,133 @@
+# typed: strict
+
+require 'cpf_cnpj'
+require 'securerandom'
+require 'net/http'
+require 'date'
+require 'uri'
+require 'json'
```

Comment:
I think some of the `require`s are not used: `json`, `date`, `securerandom`, and `net/http`.